Source/WebKit/WebProcess/WebPage/WebPage.messages.in

```diff
@@ -721,5 +721,7 @@ GenerateSyntheticEditingCommand(enum:uint8_t WebKit::SyntheticEditingCommandType
     PlayAllAnimations() -> ()
 #endif
 
+    RegisterRedirectionToFollow(WebCore::ResourceResponse response);
```

Comment:
This is a poor name.  Just looking at this message I have no idea what it is intended to be used for, what the response represents, etc.

Reply:
Changed to useRedirectionForCurrentNavigation